src/accumulator.jl

```diff
@@ -41,6 +41,8 @@ values(ct::Accumulator) = values(ct.map)
 
 sum(ct::Accumulator) = sum(values(ct.map))
 
+most_common(ct::Accumulator) = sort(collect(ct.map), by=p->p[2], rev=true)
```

Comment:
this is fine and seems like it works but I wonder if we could do this better especially for larger objects. Also this isn't really picking the "most common" elements but more just sorting the elements in order of occurrence. 

I wonder if it would be more useful and potentially faster if instead we had both this function definition plus a function definition that took in a number of elements to return so that you could get the k most common elements. I feel like this would be more useful and we could use order statistics to implement a solution with `O(n)` runtime instead of an `O(nlogn)` solution. Any thoughts on this?